### PR TITLE
android/ui: distinguish Tailnet domains in user view

### PR DESCRIPTION
**Pull request by oxtoacart**
Updates tailscale/corp#18202

### CODE REVIEW

File: android/src/main/java/com/tailscale/ipn/ui/view/UserView.kt [hunk 1]
```diff
@@ -40,10 +41,12 @@ fun UserView(
           headlineContent = {
             Text(
                 text = profile.UserProfile.DisplayName,
-                style = MaterialTheme.typography.titleMedium)
+                style = MaterialTheme.typography.titleMedium.short)
```
Comment (oxtoacart) [inline android/src/main/java/com/tailscale/ipn/ui/view/UserView.kt:44]: Drive-by formatting fix

File: android/src/main/java/com/tailscale/ipn/ui/view/MainView.kt [hunk 2]
```diff
@@ -218,7 +207,7 @@ fun ExitNodeStatus(navAction: () -> Unit, viewModel: MainViewModel) {
 }
 
 @Composable
-fun SettingsButton(user: IpnLocal.LoginProfile?, action: () -> Unit) {
```
Comment (oxtoacart) [inline android/src/main/java/com/tailscale/ipn/ui/view/MainView.kt:221]: Drive-by, unused parameter

**Review by barnstar — APPROVED** [review]
Functionally, this is fine... But it's different.  It does feel more correct.  Design should really weigh in here.

**Comment by oxtoacart** [comment]
Thanks @barnstar , I'll merge and show to @sonovawolf in our walkthrough tomorrow.
